hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/hudi/DataSourceOptions.scala

```diff
@@ -306,6 +306,11 @@ object DataSourceWriteOptions {
     .withDocumentation("When set to true, will perform write operations directly using the spark native " +
       "`Row` representation, avoiding any additional conversion costs.")
 
+  val SQL_CREATE_DATABASE_TABLE_NAME: ConfigProperty[String] = ConfigProperty
+    .key("hoodie.sql.create.database.table.name")
```

Comment:
making the purpose clearer 

```suggestion
    .key("hoodie.sql.uses.database.table.name")
```

Reply:
Hello, I have modified it